doc/source/tune/api_docs/internals.rst

```diff
@@ -12,6 +12,14 @@ Main Components
 
 Tune's main components consist of TrialRunner, Trial objects, TrialExecutor, SearchAlg, TrialScheduler, and Trainable.
 
+.. _trial-runner-flow:
+
+This is an illustration of the high-level training flow and how some of the components interact:
+
+.. figure:: ../../images/tune-trial-runner-flow-horizontal.png
```

Comment:
nice! Could you add a small legend to this? (yellow, blue, green ,etc)

It'd also be nice to point out that this figure is 'scrollable'